build.psm1

```diff
@@ -2754,7 +2762,9 @@ function Start-CrossGen {
         )
     }
 
-    foreach ($assemblyName in $psCoreAssemblyList) {
+    $fullAssemblyList = $commonAssembliesForAddType + $psCoreAssemblyList
+
+    foreach ($assemblyName in $fullAssemblyList) {
```

Comment:
Can this loop be merged with loop in L2779? So we iterate over the list only once.